.circleci/config.yml

```diff
@@ -162,31 +162,78 @@ jobs:
           - ~/.m2
       - <<: *push_images
 
-  # Connector depends on integration, mount workspace .m2
-  connector:
+  # Connector base support depends on integration, mount workspace .m2
+  connector-base:
     <<: *job_defaults
     environment:
       <<: *common_env
     steps:
       - checkout
       - restore_cache:
-          key: syndesis-mvn-connector-{{ checksum "app/connector/pom.xml" }}
+          key: syndesis-mvn-connector-{{ checksum "app/connector-support/pom.xml" }}
       - <<: *load_m2
       - run:
-          name: Build Connectors
+          name: Build Connector Util
```

Comment:
Perhaps we can introduce two modules one containing all support modules for connectors and one containing only connectors. Would make these bits a bit simpler.

Reply:
I have improved a bit to run all modules in one run command. Also it is a bit more complicated than I thought. We need to build some support modules first then all connectors and at the very end the connector-catalog.